src/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProvider.php

```diff
@@ -262,6 +265,37 @@ private function extractRelatedProducts(int $productId): array
         return $relatedProductsData;
     }
 
+    /**
+     * @param int $productId
+     *
+     * @return array<int, array<string, int|string>>
+     */
+    private function extractPackedProducts(int $productId): array
+    {
+        /** @var PackedProductDetails[] $packedProductsDetails
+         */
+        $packedProductsDetails = $this->queryBus->handle(
+            new GetPackedProductsDetails(
+                $productId,
+                $this->contextLangId
+            )
+        );
+        $packedProductsData = [];
+        foreach ($packedProductsDetails as $packedProductDetails) {
+            $productName = $packedProductDetails->getProductName();
+            $packedProductsData[] = [
+                'id' => $packedProductDetails->getProductId(),
```

Comment:
I would say its worth to name this `id` to `productId` as its not so obvious what it actually is (especially when we also have combinationId and even some kind of unique id built from both of them as you called uniqueIdentifier - which could also be simplified to uniqueId).
Also one more thing, I wonder if we shouldn't stick to snake_case here? most of the time we use snake case everywhere (especially in forms). Only exceptions when we need to provide json data with some api or so.. ping @jolelievre

Reply:
It makes sense. I also like to have significant names. I will investigate about the side effects.